packages/trader/src/Modules/CFD/Containers/cfd-dashboard.jsx

```diff
@@ -15,7 +15,7 @@ import { ResetTradingPasswordModal } from '@deriv/account';
 import { connect } from 'Stores/connect';
 import LoadingCFDRealAccountDisplay from './loading-cfd-real-account-display.jsx';
 import MissingRealAccount from './missing-real-account.jsx';
-import MT5AccountOpeningRealFinancialStpModal from './mt5-account-opening-real-financial-stp-modal.jsx';
+import MT5AccountOpeningRealFinancialStpModal from './mt5-account-opening-real-financial-stp-modal';
```

Comment:
I see such `.jsx` ending in every other file imported. Why is this different? Is this intentional?

Reply:
It is the way that all of our components are imported. Since we are doing the refactoring, I change the imports just for the components that are changed to TypeScript and leave our JS component same as before.